src/Components/Members/Table/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```

Comment:
We were told to put everything in the same return, from and table should be in the Members index